lua/nvim-treesitter/highlight.lua

```diff
@@ -98,18 +97,14 @@ function M.attach(bufnr, lang)
     hlmap[k] = v
   end
 
-  local query = queries.get_query(lang, "highlights")
-  if not query then return end
-
-  M.highlighters[bufnr] = ts.highlighter.new(parser, query)
+  ts.highlighter.new(parser, {})
 end
 
 function M.detach(bufnr)
-  if M.highlighters[bufnr] then
-    M.highlighters[bufnr]:set_query("")
-    M.highlighters[bufnr] = nil
+  if ts.highlighter.active[bufnr] then
+    ts.highlighter.active[bufnr]:destroy()
```

Comment:
thanks for fixing this issue.

Reply:
I waiting so long to fix it nobody should ever thank me :wink: